packages/project/src/db/props.ts

```diff
@@ -1,22 +1,73 @@
 import {
   type Tree,
   type AllUserProps,
-  UserProps,
+  type UserProp,
+  // TODO: move zod schema into project package
```

Comment:
forgotten? or wanted to do with a separate PR? 
inn any case we use the `@todo` style comment

Reply:
Separate